pkg/metadata/inventories/README.md

```diff
@@ -93,6 +93,7 @@ The payload is a JSON dict with the following fields
     `network_config.enabled` config option in `system-probe.yaml`).
   - `feature_networks_http_enabled` - **bool**: True if HTTP monitoring is enabled for Network Performance Monitoring (see: `network_config.enable_http_monitoring` config option in `system-probe.yaml`).
   - `feature_networks_https_enabled` - **bool**: True if HTTPS monitoring is enabled for Network Performance Monitoring (see: `network_config.enable_https_monitoring` config option in `system-probe.yaml`).
+  - `feature_remote_configuration_enabled` - **bool**: True if the Cloud Workload Security is enabled (see: `remote_configuration.enabled` config option).
```

Comment:
Is it really supposed to be `Cloud Workload Security` here ?

Reply:
Yup, that's fixed, thanks!